business/metrics.go

```diff
@@ -226,3 +226,31 @@ func createStatsMetricsLabelsBuilder(q *models.MetricsStatsQuery) *MetricsLabels
 	}
 	return lb
 }
+
+func (in *MetricsService) GetControlPlaneMetrics(q models.IstioMetricsQuery, scaler func(n string) float64) (models.MetricsMap, error) {
+	metrics := make(models.MetricsMap)
+
+	h := in.prom.FetchHistogramRange("pilot_proxy_convergence_time", "", "", &q.RangeQuery)
+	var err error
+	converted, err := models.ConvertHistogram("pilot_proxy_convergence_time", h, models.ConversionParams{Scale: 1})
+	if err != nil {
+		return nil, err
+	}
+	metrics["pilot_proxy_convergence_time"] = append(metrics["pilot_proxy_convergence_time"], converted...)
+
+	metric := in.prom.FetchRateRange("process_cpu_seconds_total", []string{`{app="istiod"}`}, "", &q.RangeQuery)
```

Comment:
The `process_cpu_seconds_total` metric is not on the [required metric list](https://kiali.io/docs/faq/general/#requiredmetrics) and needs to be added.   It is not in our [thinning config](https://kiali.io/docs/configuration/p8s-jaeger-grafana/prometheus/#metric-thinning), and so no changes are required there.